modules/pallets/ismp/src/host.rs

```diff
@@ -314,6 +316,10 @@ impl<T: Config> IsmpHost for Pallet<T> {
 		Responded::<T>::insert(req_commitment, true);
 		Ok(())
 	}
+
+	fn previous_commitment_height(&self, id: StateMachineId) -> Result<u64, Error> {
+		Ok(PreviousStateMachineHeight::<T>::get(id).unwrap_or_default())
```

Comment:
Maybe lets make this function return an option instead since the underlying storage is option query